src/components/Text/Text.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./Text.module.scss";
+import { PropTypes } from "react";
+
+const sizeToClass = {
+  h1: styles.text_h1,
+  h2: styles.text_h2,
+  h3: styles.text_h3,
+  h4: styles.text_h4,
+  p: styles.text_paragraph,
+};
+
+export function Text({ text, size = "p" } = {}) {
+  const Component = size.toLowerCase() || "p";
```

Comment:
The old chinese proverb says - `Better safe than... RuntimeError` 😄 
What if someone would pass `"czupakabra"` as a value for `size` prop? I think that it may go kaboom 😉 
Maybe it would be safer to go with an object including predefined set of keys (`h1`, `h2`, ...) and we could do smth like `sizeToComponent[size] || "p"` to have a fallback to `p` if the key is not found?